components/text.js

```diff
@@ -129,7 +145,14 @@ export default memo(function Text ({ nofollow, imgproxyUrls, children, tab, ...o
             // we don't render it as an image since it was probably a concious choice to include text.
             const text = children[0]
             if (!!text && !/^https?:\/\//.test(text)) {
-              return <a target='_blank' rel={`noreferrer ${nofollow ? 'nofollow' : ''} noopener`} href={href}>{text}</a>
+              // Render video if it's a valid Youtube url, otherwise null
+              const video = renderYoutubeUrl(href);
+              return (
+                <>
+                  <a target='_blank' rel={`noreferrer ${nofollow ? 'nofollow' : ''} noopener`} href={href}>{text}</a>
+                  {video}
```

Comment:
Won't this show the link _and_ the embedded video player?

Reply:
> Btw do you know if the hot reload is working? Seems like I'll have to restart docker container every time after I made changes to see the update, and it takes a couple of minutes. Thanks!

Mhh, it should work for the app but I am using `npm run dev` now. When I was using docker, hot reload worked.

Reply:
No worries. I think huumn has taken care of it. And thanks for the info